src/qt/splashscreen.cpp

```diff
@@ -64,7 +64,7 @@ SplashScreen::SplashScreen(interfaces::Node& node, Qt::WindowFlags f, const Netw
     pixPaint.fillRect(rGradient, gradient);
 
     // draw the bitcoin icon, expected size of PNG: 1024x1024
```

Comment:
```suggestion
    // draw the bitcoin corr icon, expected size of PNG: 1024x1024
```

Reply:
@jeandudey Please note that the suggested name is "Corrr" (three R) and not "Corr" (two R).

It might not sound important but the third R really underscores that the money printer is running at full capacity.

Somewhat related: "corr" might be misinterpreted as "correlation" and it should be noted that correlation does not imply causation. Correlation is fragile. Causation is robust (like Bitcoin Corrr).

Reply:
@jeandudey Please note that the suggested name is "Corrr" (three R) and not "Corr" (two R).

It might not sound important but the third R really underscores that the money printer is running at full capacity.

Somewhat related: "corr" might be misinterpreted as "correlation" and it should be noted that correlation does not imply causation. Correlation is fragile. Causation is robust (like Bitcoin Corrr).

Reply:
Sorry my mistake, this isn't a blocker for me, of course the correct "Corrr" should be added if it can be fixed ASAP.

Not a strong opinion though.